server/policy_test.go

```diff
@@ -151,3 +166,113 @@ func TestEvaluatePolicyFailure(t *testing.T) {
 		})
 	}
 }
+
+func TestSevSnpSignedUefi(t *testing.T) {
+	// Generate fake signed reference measurements.
+	fw := ovmfsev.CleanExample(t, 0x200000)
+	meas, err := gcesev.LaunchDigest(gcesev.LaunchOptionsDefault(), fw)
+	if err != nil {
+		t.Fatal(err)
+	}
+	certdir := t.TempDir()
+	keydir := t.TempDir()
+	outdir := t.TempDir()
+	outdir2 := t.TempDir()
+	if err := devkeys.DumpTo(&devkeys.Options{
+		KeyDir:   keydir,
+		CertRoot: certdir,
+		CertDir:  "signer_certs",
+		Bucket:   "certs-dev",
+	}); err != nil {
+		t.Fatal(err)
+	}
+	fwdir := t.TempDir()
+	fwPath := path.Join(fwdir, "fw.fd")
+	if err := os.WriteFile(fwPath, fw, 0666); err != nil {
+		t.Fatal(err)
+	}
+
+	nonprod.RootCmd.SetArgs([]string{"endorse",
+		"--uefi", fwPath,
+		"--key_dir", keydir,
+		"--bucket_root", certdir,
+		"--root_path", "root.crt",
+		"--add_snp",
+		"--out_root", outdir,
+		"--snapshot_dir", "snap",
```

Comment:
Seems a bit odd that we have to test through the CLI instead of through an underlying library

Reply:
It's a CLI library. Using it this way is less boilerplate-y.